<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorSSR</class>
 <widget class="QWidget" name="InspectorSSR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Screen Space Reflections (SSR)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Distance</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Roughness Mask</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_pIntensityEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Intensity</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Double Reflections</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_pRoughnessMaskEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="m_pDistanceEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="m_pDoubleReflectionsCB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pIntensityEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorSSR</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRoughnessMaskEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorSSR</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pDistanceEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorSSR</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pDoubleReflectionsCB</sender>
   <signal>clicked()</signal>
   <receiver>InspectorSSR</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valueChanged()</slot>
 </slots>
</ui>
